packages/scaffold/src/views/w3m-connecting-social-view/index.ts

```diff
@@ -163,6 +163,14 @@ export class W3mConnectingSocialView extends LitElement {
   }
 
   private connectSocial() {
+    const interval = setInterval(() => {
+      if (this.socialWindow?.closed) {
+        if (!this.connecting && RouterController.state.view === 'ConnectingSocial') {
+          RouterController.goBack()
+        }
+        clearInterval(interval)
```

Comment:
I'm confused how this is working in terms of closures. The callback function is not binded to the parent class and I wouldn't expect the `interval` value is available inside the `setInterval`'s callback function. Weird, let's double check this in case it'd be problematic in different environments. Smells to me but cannot be sure tbh